v2.1/export.md

```diff
@@ -130,6 +130,24 @@ Convert SQL *NULL* values so they match the specified string.
 $ cockroach sql -e "SELECT * from bank.customers WHERE id>=100;" --format=csv > my.csv
 ~~~
 
+### View an export
```

Comment:
a *running* export

unlike jobs, show queries is just while it is running.

ditto below.